modules/apps/frontend-js/frontend-js-web/test/liferay/util/format_storage.es.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+import formatStorage from '../../../src/main/resources/META-INF/resources/liferay/util/format_storage.es';
+
+describe('Liferay.Util.formatStorage', () => {
+	it('should return formatted string with size and suffix type KB if size input type is number <= 1048575 and options is default value', () => {
+		expect(formatStorage(10400)).toEqual('10KB');
+	});
+
+	it('should return formatted string with negative size and suffix type KB if size input type is negative number and options is default value', () => {
+		expect(formatStorage(-10400)).toEqual('-10KB');
```

Comment:
This is silly behavior, negative size?? Either remove this test, or throw error in the code when `size` is negative, testing that behavior.